package.json

```diff
@@ -19,22 +19,17 @@
     "url": "https://github.com/edenspiekermann/a11y-dialog"
   },
   "files": [
-    "a11y-dialog.js",
-    "a11y-dialog.min.js",
-    "a11y-dialog.d.ts"
+    "dist/a11y-dialog.js",
+    "dist/a11y-dialog.min.js",
+    "dist/a11y-dialog.d.ts"
```

Comment:
Those three lines could be replaced by `"dist/*"`.
(Tested locally with `npm pack`.)